examples/k8s_glue_example.py

```diff
@@ -91,7 +95,10 @@ def k8s_user_props_cb(pod_number=0):
             ssh_port_number=args.ssh_server_port) if args.ssh_server_port else None,
         namespace=args.namespace, max_pods_limit=args.max_pods or None,
     )
-    k8s.k8s_daemon(args.queue, use_owner_token=args.use_owner_token)
+    if "," in args.queue:
```

Comment:
This seems too elaborate, you can simply do:
```python
args.queue = [q.strip() for q in args.queue.split(",") if q.strip()]
```
Which would also handle input such as `"queue1,"`